src/KeysManager.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\pubkey_encrypt\KeysManager.
+ */
+
+namespace Drupal\pubkey_encrypt;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\user\PrivateTempStoreFactory;
+use Drupal\user\UserInterface;
+
+/**
+ * Handles users' Public/Private key pairs.
+ */
+class KeysManager {
```

Comment:
This might not be the plugins used, so I still think something with public key encryption in he name is more appropriate